src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -73,6 +73,14 @@ The `type` and default key is `"device"`.
 
 : _Optional_. The screen resolution. (e.g.: 800x600, 3040x1444).
 
+`screen_height_pixels`
+
+: _Optional_. The height of the screen.
```

Comment:
Ideally this would be explicit about the unit, and/or an example.And clarify how does it relate to `screen_resolution`?

If `screen_resolution` is 800x600 this would be `800` and `screen_width_pixels` would be `600`?
I wonder if any SDK sends this already, couldn't find this key in the Java/Android or Cocoa SDKs

Reply:
Android SDK was previously sending `screen_resolution` but we added screen_width_pixels and screen_height_pixels, because we also send screen_density and screen_dpi, I don't remember the reason why we've split up, maybe because they were read from separate fields but required a ValueXValue format in the protocol so we needed to string format to do calculations

Reply:
I'm also never sure if it's width times height or the other way around so I really appreciate two properties which state exactly what they're for.

Reply:
maybe that was the reason :D